avm/res/synapse/workspace/tests/e2e/defaults/main.test.bicep

```diff
@@ -56,8 +56,5 @@ module testDeployment '../../../main.bicep' = [
       defaultDataLakeStorageFilesystem: nestedDependencies.outputs.storageContainerName
       sqlAdministratorLogin: 'synwsadmin'
     }
-    dependsOn: [
```

Comment:
Something I also did in the latest updates was to remove the `location` property from all tests but the max test. The original linter rule that caused us to add it is not active anymore.

Reply:
done